docs/calculations.mdx

```diff
@@ -270,16 +271,22 @@ Account for non-required fields:
       default_dynamic_watts_per_kbps[network_type] :
       average_dynamic_watts_per_kbps
 
-  is_audio_creative =
-      creative_audio_data_transfer_bytes ||
-      creative_audio_duration_seconds ||
-      ad_format.audio_duration_seconds
-  default_channel =
-      is_audio_creative ?
-      audio :
-      default_channel_by_device[device_type]
-  channel = channel ? default_channel
-
+  lookup_channel:
+    if property.channels.length == 1:
```

Comment:
What about the case when we don't have a property (and channel and device)? eg when calculating seller emissions

Reply:
I think that case is out of scope of this repo - clarified that property is a required object here. In general I think that if you're just specifying seller you should have to also include channel, or even an array of channels to consider, given the breadth of multi-channel media companies. I guess if somebody puts in Meta you could see that all of their properties are social, but also reasonably to just recurse through their properties and calc at the property level.